.ki-scripts/startup.js

```diff
@@ -411,31 +432,48 @@ pm2.connect(false, async function (err) {
   // ------------- TESTNET ENVIRONMENT STARTUP ------------------
 
   if (process.env.CHAIN_ENV === "testnet") {
-    const USE_EXISTING = jetpack.exists(".env.testnet");
+    if (process.env.E2E_TESTING === "true") {
+      // E2E test run is only for github actions when pushed
+      spinner.warn(
+        "Using existing testnet account credentials for e2e testing..."
+      );
+
+      dotenv.config({
+        path: requireEnv(process.env.CHAIN_ENV),
+      });
+
+      // For E2E testing, we only need to deploy and initialize account
+      // We don't need to run cleanup or set up new account because we are using a pre-existing
+      // testing account
 
-    if (!USE_EXISTING) {
-      await cleanupTestnetConfig();
-      await bootstrapNewTestnetAccount();
       await deployAndInitialize();
     } else {
-      let useExisting = await inquirer.prompt({
-        type: "confirm",
-        name: "confirm",
-        message: `Use existing tesnet credentials in ${chalk.greenBright(
-          "env.testnet"
-        )} ?`,
-        default: true
-      });
+      const USE_EXISTING = jetpack.exists(".env.testnet");
 
-      if (!useExisting.confirm) {
-        spinner.warn("Creating new testnet account credentials...");
+      if (!USE_EXISTING) {
         await cleanupTestnetConfig();
         await bootstrapNewTestnetAccount();
         await deployAndInitialize();
       } else {
-        dotenv.config({
-          path: requireEnv(process.env.CHAIN_ENV)
+        let useExisting = await inquirer.prompt({
```

Comment:
interact with the prompt instead of adding code change paths

Reply:
Refactored this part of the code, as there was no clear solution to interacting with the prompt in Github actions + cypress